src/useMintUsd.ts

```diff
@@ -79,7 +79,7 @@ export function useMintUsd({
         CoreProxyContract,
         accountId,
         poolId,
-        tokenAddress: collateralTokenAddress,
+        tokenAddress: collateralTypeTokenAddress,
```

Comment:
```suggestion
        collateralTypeTokenAddress,
```

Worth updating fetchers too to maintain the same naming.

Reply:
Done, in those fetchers where these cases happened. We need to discuss about the remaining cases.